test/1.4/legacy/T_shared_logs_on_device_disabled.dml

```diff
@@ -0,0 +1,44 @@
+/*
+  © 2023 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+
+device test;
+
+/// DMLC-FLAG --simics-api=6
```

Comment:
Note that this will give a compile error when building DMLC for simics 8; someday we'll need to add an annotation to handle that. Today is not that day, though.

Reply:
in fact this line is redundant; what we need is a test that --simics-api=6 *without* passing `--no-compat` gives the buggy behaviour. It is that test that will be problematic for simics 8.

Reply:
and now I see you wrote that test already, [T_shared_logs_on_device_enabled.dml](https://github.com/intel/device-modeling-language/pull/225/files#diff-1d79db2582c21fd49607b71ae51a13e6c26b798a14380455525b3abe780dc58b).

Reply:
You're complaining about this line, but I just followed the pattern of the legacy tests _you_ created!

Reply:
Right! so whose fault is it that this mistake of mine wasn't caught in YOUR review of that code? :)